vochain/keykeeper/keykeeper.go

```diff
@@ -433,12 +448,21 @@ func (k *KeyKeeper) publishKeys(pk *processKeys, pid string) error {
 	k.lock.Lock()
 	defer k.lock.Unlock()
 	dbKey := []byte(dbPrefixProcess + pid)
-	// TODO(mvdan): replace Has with just Get
-	if exists, err := k.storage.Has(dbKey); exists || err != nil {
-		return fmt.Errorf("keys for process %x already exist or error fetching storage: (%s)", pid, err)
+
+	wTx := k.storage.WriteTx()
+	defer wTx.Discard()
+
+	_, err := wTx.Get(dbKey)
```

Comment:
`if _, err := wtx.Get(dbKey); err == db.ErrKeynotFound {`

Reply:
Ok, but outside the if, there is this line:
```go
return fmt.Errorf("keys for process %x already exist or error fetching storage: (%s)", pid, err)
```
I've added a `var err error` before this line, so the error can be returned later outside this if.